docs/03_tutorials/20_tic-tac-toe-game-smart-contract-single-node.md

```diff
@@ -0,0 +1,765 @@
+
+
+We explain the purpose of each step and why it is important.
+
+## Prerequisites for the Tutorial
+This tutorial requires the following tools and knowledge:
+
+* EOSIO platform software, specifically `cleos`, `keosd` and `eosio.cdt` which compiles the smart contract. Click on this link to [Install the CDT.](https://developers.eos.io/welcome/latest/getting-started/development-environment/install-the-CDT "EOSIO.CDT Installation Instructions") Click on this link for an [overview of the EOSIO platform and toolchain.](https://developers.eos.io/welcome/latest/overview/platform_and_toolchain)
+* Knowledge of the C++ programming language
+* A code editor or an IDE. Click on this link for more information [EOSIO Prerequisites.](https://developers.eos.io/welcome/latest/getting-started/development-environment/introduction "Set up to use EOSIO")
+   
+### Tutorial Workflow
+
+```sequence
+Understand\n Game->Understand\n EOSIO : What?
+Understand\n EOSIO->Create\nAccounts  : Accounts
+Understand\n EOSIO->Create\nAccounts  : Actions
+Understand\n EOSIO->Create\nAccounts  : Resources
+Create\nAccounts->Create\nSmart Contract : How?
+Create\nSmart Contract -> Compile\nDeploy : Code
+Compile\nDeploy -> Play\nGame : Load
+Play\nGame -> Play\nGame : Push
+Play\nGame -> Play\nGame : Get
+```
+
+## The Rules of the Game
+The EOSIO tic-tac-toe rules are:
+
+* Player one (the host) makes the first move, followed by player two (the challenger). 
+* The first player to complete a row or diagonal of either X’s or O’s wins the game.
+* If no player completes a row or diagonal of either X’s or O’s, the game is a draw. 
+
+## Understand how the game works
+The smart contract contains the game logic, therefore this section introduces how the game works and some of the EOSIO concepts needed to build a smart contract.
+
+The game is played by two players, so we need two blockchain accounts. This tutorial explains how to create and use these blockchain accounts on a local single node blockchain in the next step. Use these accounts to "push actions" to the blockchain. These actions start, restart, and close the game. Pushing a move action will place a 'marker' on the game 'board'.    
+
+One player is the host, who starts the game, and one player is the challenger. The game board is nine squares, and each player takes a turn to place their marker in a square. A player wins the game when three markers are placed in a row.
+
+In the example below the player who placed x has won.
+
+| Row       | 0     | 1     | 2     |
+| :-------: | :---: | :---: | :---: |
+|     0     |   -   |   o   |   x   |
+|     1     |   -   |   x   |   -   |
+|     2     |   x   |   o   |   o   |
+
+When all the squares contain a marker and no player has three markers in a row, then the game is a draw.
+
+### Accounts, key pairs, and secure your private key
```

Comment:
This heading is too long and is getting truncated in the right navigation (tested locally).

Suggestion:

### Accounts and Key Pairs

Securing private keys can be explained in the para or a callout note.